### ISSUE TITLE
Using autoWidth:true, loop:true, and autoplay:true. Gaps at end of loop.

### ISSUE THREAD

**Issue by 57u** [smashingboxes/OwlCarousel2#697]
loop doesn't seem to work correctly when using autoWidth:true, loop:true, and autoplay:true together, see this fiddle: http://jsfiddle.net/yau5fy66/ 

When getting to the end, theres a gap, few seconds, and then jumps into place again...

**Comment by Anydog** [comment]
Yes, I can report this too ... I thought at first that it was an code optimization issue (cloning items is probably "heavy job" fo browser), but it seems more like  bug ... :(

**Comment by Dakshank** [comment]
while loop is false; also slider not working well, as last item should not have space at the end 
http://jsfiddle.net/yau5fy66/6/
please fix the bug.
thank you.

**Comment by Dakshank** [comment]
for the above problem
/*.owl-carousel .owl-item img {
  transform-style: preserve-3d;
}*/
.owl-carousel .owl-item img {
    display: block;
    width: 100%;
    /*-webkit-transform-style: preserve-3d;*/
}
comment those styles will solve the problem 
as i was did the same for our project.

thank you.

**Comment by parhumm** [comment]
I have this problem too.

**Comment by junnaa** [comment]
We have this problem too. fix plz.

**Comment by Samuel-Weiss** [comment]
the same here. Quick and dirty try change the following line 
Owl carousel @version 2.0.0

Line 289:
`delta = this.settings.loop ? clones.length - Math.max(this.settings.items * 2, 4) : 0;`
`delta = this.settings.loop ? clones.length - Math.max(this.settings.items * 2, 18) : 0;`

**Comment by nroyall** [comment]
Also noticing this issue, with autoplay set to the default setting (false) and autoWidth:true, loop:**false**.
